goss-testing/tests/common/goss-network-interfaces.yaml

```diff
@@ -29,7 +29,13 @@ command:
     meta:
       desc: Ensures VLAN was created for network interface '{{$interface}}'.
       sev: 0
-    exec: 'ip link show dev {{$interface}}'
+    # For bond0.can0, if it does not exist, we do not fail. Echo a fake IP so the test passes
+    exec: |-
```

Comment:
This has the same effect as just not running the test for bond0.can0, since it will never fail for that interface. If that is the desired behavior, may as well just skip the test for bond0.can0 using goss conditional logic, IMO.

Reply:
Forgot to push here.  Check now

Reply:
This still will never fail for bond0.can0

Reply:
I would change it to something like:
```
command:
    {{range $interface := index .Vars "network_interfaces"}}
        {{if ne {{$interface}} "bond0.can0"}}
            check_interface_vlan_{{$interface}}:
                title: Interface '{{$interface}}' Has VLAN
                meta:
                    desc: Ensures VLAN was created for network interface '{{$interface}}'.
                    sev: 0
                exec: 'ip link show dev {{$interface}}'
                exit-status: 0
                timeout: 10000
                skip: false
        {{end}}
    {{end}}
```

Reply:
That seems to work!

Reply:
Fixed